frontend-new/src/pages/index.vue

```diff
@@ -89,84 +91,48 @@ const licenses = [
-            <input :id="version.version" :name="version.version" type="checkbox" class="hidden" />
-            <span
-              class="absolute top-5px left-0 h-15px w-15px rounded-sm bg-white group-hover:(bg-gray-300) after:(absolute hidden content-DEFAULT top-3px left-6px w-5px h-10px border-solid border-r-3px border-b-3px)"
-            />
-          </label>
+        <div class="flex flex-col gap-2 max-h-30 overflow-auto">
+          <LabeledCheckbox v-for="version in versions" :key="version.version" :label="version.version" />
         </div>
       </div>
       <hr />
       <div class="categories">
         <h3 class="font-bold">Categories</h3>
         <div class="flex flex-col gap-2">
-          <label v-for="category in categories" :key="category.id" class="group relative cursor-pointer pl-30px customCheckboxContainer">
-            {{ category.label }}
-            <input type="checkbox" class="hidden" />
-            <span
-              class="absolute top-5px left-0 h-15px w-15px rounded-sm bg-white group-hover:(bg-gray-300) after:(absolute hidden content-DEFAULT top-3px left-6px w-5px h-10px border-solid border-r-3px border-b-3px)"
-            />
-          </label>
+          <LabeledCheckbox v-for="category in categories" :key="category.id" :label="category.label" />
         </div>
       </div>
       <hr />
       <div class="platforms">
         <h3 class="font-bold">Platforms</h3>
         <div class="flex flex-col gap-2">
-          <label v-for="platform in platforms" :key="platform.id" class="group relative cursor-pointer pl-30px customCheckboxContainer">
-            {{ platform.label }}
-            <input type="checkbox" class="hidden" />
-            <span
-              class="absolute top-5px left-0 h-15px w-15px rounded-sm bg-white group-hover:(bg-gray-300) after:(absolute hidden content-DEFAULT top-3px left-6px w-5px h-10px border-solid border-r-3px border-b-3px)"
-            />
-          </label>
+          <LabeledCheckbox v-for="platform in platforms" :key="platform.id" :label="platform.label" />
         </div>
       </div>
       <hr />
       <div class="licenses">
         <h3 class="font-bold">Licenses</h3>
         <div class="flex flex-col gap-2">
-          <label v-for="license in licenses" :key="license.id" class="group relative cursor-pointer pl-30px customCheckboxContainer">
-            {{ license.label }}
-            <input type="checkbox" class="hidden" />
-            <span
-              class="absolute top-5px left-0 h-15px w-15px rounded-sm bg-white group-hover:(bg-gray-300) after:(absolute hidden content-DEFAULT top-3px left-6px w-5px h-10px border-solid border-r-3px border-b-3px)"
-            />
-          </label>
+          <LabeledCheckbox v-for="license in licenses" :key="license.id" :label="license.label" />
         </div>
       </div>
     </div>
   </div>
 </template>
 
 <route lang="yaml">
-meta:
-  layout: default
+meta:layout: default
```

Comment:
smth odd happened here. In intellij, alt+enter (or bring up context actions) and inject the YAML language here. That should relieve some of the formatting issues.

Reply:
I remember I had to put it in one line because otherwise it threw some error there, but I don't remember exactly. Anyways that inject yaml language thingy fixed it!